### ISSUE TITLE
Punctuation model no longer available

### ISSUE THREAD

**Issue by shammyfiveducks** [mkiol/dsnote#121]
Cannot download from huggingface  as model looks to have been removed.

**Comment by mkiol** [comment]
Many thanks for reporting. Indeed, it was removed.

I re-uploaded this model to my own repo here: https://huggingface.co/mukowaty/punctuate-16.

Fix: 7574aa5216f02a6d1bc2e91ed811393a45ca0361

If you want to fix this without waiting for the next release, you can edit `~/.var/app/net.mkiol.SpeechNote/data/net.mkiol/dsnote/models.json` file and update urls manually for model `multilang_hftc_kredor`.

```
      {
          "name": "Punctuation",
          "model_id": "multilang_hftc_kredor",
          "engine": "ttt_hftc",
          "lang_id": "multilang",
          "checksum": "5294921e",
          "checksum_quick": "b8b03e8a",
          "comp": "dir",
          "urls": [
              "https://huggingface.co/mukowaty/punctuate-16/resolve/3eaff4f3669ab76b3f8b6c33aadcae4a4399e311/config.json",
              "https://huggingface.co/mukowaty/punctuate-16/resolve/3eaff4f3669ab76b3f8b6c33aadcae4a4399e311/pytorch_model.bin",
              "https://huggingface.co/mukowaty/punctuate-16/resolve/3eaff4f3669ab76b3f8b6c33aadcae4a4399e311/tokenizer.json"
          ],
          "size": "1126993609",
          "hidden": true
      },
```

**Comment by mkiol** [comment]
New version 4.5.0 is out. The fix is included.
